src/main/resources/config.yml

```diff
@@ -0,0 +1,14 @@
+MarkerSetId: "offplrs"
+MarkerSetName: "Offline Players"
+
+#Whether to use bluemaps own player icons instead of downloading manuelly
```

Comment:
typo: `#Whether to use BlueMap's own player icons, instead of downloading manually`